cypress/integration/existing_user/logged_in.spec.js

```diff
@@ -0,0 +1,50 @@
+import {
+  expandTocItem,
+} from '../../support/utils'
+
+describe('Logged In User', () => {
+  beforeEach(() => {
+    cy.login_request(Cypress.env('JOURNAL_USER_EMAIL'), Cypress.env('JOURNAL_USER_PASSWORD'))
```

Comment:
I think we should have `login` command. So instead writing these 4  statements we should just have single statement to login a user here.

Reply:
hmm yes that would look better, it's only that the actual login is done for courses.stage.edx.org, the next commands are using this session cookies to login to journal, so there are two distinct operations being performed here, thats why I haven't put all these statements in login command  but I am definitely open to suggestions